toast-tk-runtime/src/main/java/io/toast/tk/runtime/result/ObjectResultHandler.java

```diff
@@ -9,10 +14,39 @@
     public ObjectResultHandler(IActionItemRepository objectRepository) {
         super(objectRepository);
     }
-    
+
+	@SuppressWarnings({ "unchecked", "rawtypes" })
+    private String resultBuetifier(Object value) {	
+		try {
+			return ((List)value).stream()
+					.map(Object::toString)
+					.collect(Collectors.joining(System.lineSeparator()))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "collect" returns a string, there's no need to call "toString()". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1858)